other-docs/guides/migrating-from-wordpress.md

```diff
@@ -0,0 +1,150 @@
+# Migrating a WordPress Codebase
+
+This guide covers how to migrate a typical WordPress project to HM Platform.
+
+### Setup Composer
+
+If the project does not already have a `composer.json` in the root of the repository, you should run `composer init` from the project root. This will prompt for any fields. For now, don’t bother adding any dependancies. If you already have a `composer.json` move to Remove Dependencies.
+
+### Remove Dependencies
+
+We’ll need to remove all the dependencies that are part of HM Platform, as those will be installed by HM Platform itself. Firstly, remove the any submodules that are included in HM Platform. For example, `wordpress` and `hm-platform` submodules. To correctly remove a submodule:
+
+1. `git submodule deinit -f wordpress`
+2. `git rm -rf wordpress`
+3. `rm -rf .git/modules/wordpress`
+
+Do the same for `hm-platform` if you have this project dependency. HM Platform also includes the following WordPress plugins, so if you project uses git submodules or uses composer to include them, you should remove them from your project (unless you plan to disable the HM Platform module that makes use of the plugin, and continue with the plugin directly):
```

Comment:
```suggestion
Do the same for `hm-platform` if you have this project dependency. HM Platform also includes the following WordPress plugins, so if your project uses git submodules or composer to include them, you should remove them from your project (unless you plan to disable the HM Platform module that makes use of the plugin, and continue with the plugin directly):
```

Reply:
The part in brackets doesn't really have a use case just yet and maybe adds some confusion about when you might do that. I think it could be left out of this guide myself, or maybe mentioned as a footnote with examples of when you might want to use a plugin directly over a platform module.

Also got me thinking the package list in the core module should maybe reflect which modules are active in this case as a plugin present there won't be installed to the content directory.

Reply:
Created https://github.com/humanmade/platform-cms-installer/issues/3 to track that